CMakeLists.txt

```diff
@@ -82,3 +82,11 @@ if (VOXELENGINE_BUILD_TESTS)
     enable_testing()
     add_subdirectory(test)
 endif()
+
+add_custom_target(copy_resources ALL
```

Comment:
Line 79?

Reply:
Whoops, line 79 wast copying resources for me so i added my own version of it. I'll remove that

Reply:
Fixed